tasks/eth/base-002-fp-upgrade/README.md

```diff
@@ -0,0 +1,44 @@
+# Base Mainnet Fault Proofs Upgrade
+
+Status: [READY TO SIGN]
+
+## Objective
+
+This is the playbook upgrades Base Mainnet to use the permissionless fault proofs system.
+
+The Base team has already deployed and tested the fault proof contract set. Refer to the [release notes](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.6.0) for the contract release details.
+
+Governance post of the Fault Proofs upgrade can be found at https://gov.optimism.io/t/upgrade-proposal-10-granite-network-upgrade/8733.
+
+1. Upgrades the `OptimismPortal` proxy implementation.
+2. Upgrades the `SystemConfig` proxy implementation.
+
+Details on the upgrade procedure can be found in [EXEC.md](./EXEC.md). Signers need not validate the, but they are provided for reference.
+
+## Preparing the Upgrade
+
+The contract implementations and new proxies for the Fault Proof system have been pre-deployed to mainnet.
+
+- `AnchorStateRegistry`: `0x60F1Ea7B3359a4008655df44560B6899B1877a15`,
+- `AnchorStateRegistryProxy`: `0xdB9091e48B1C42992A1213e6916184f9eBDbfEDf`,
+- `DelayedWETH`: `0x71e966ae981d1ce531a7b6d23dc0f27b38409087`
+- `DelayedWETHProxy`: ``
+- `DisputeGameFactory`: `0xc641a33cab81c559f2bd4b21ea34c290e2440c2b`
+- `DisputeGameFactoryProxy`: `0x43edb88c4b80fdd2adff2412a7bebf9df42cb40e`
+- `FaultDisputeGame`: `0xCd3c0194db74C23807D4B90A5181e1B28cF7007C`
+- `Mips`: `0x0f8EdFbDdD3c0256A80AD8C0F2560B1807873C9c`
+- `OptimismPortal2`: `0x49048044D57e1C92A77f79988d21Fa8fAF74E97e`
```

Comment:
I believe this is the existing portal

Reply:
You're right, I updated the name to OptimismPortalProxy signify it is the proxy